recipes/python-constraint2/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "python-constraint2" %}
+{% set version = "2.0.0b4" %}
```

Comment:
We prefer to have releases on conda-forge, can you please submit one of these?

Reply:
I'm not sure if I understand the comment, my apologies. Do you mean a release that is not a beta version?

If the issue is indeed that it is a beta release, the reason for doing it like this is the following: version 0.x of kernel tuner [does not include the source distribution on PyPI](https://pypi.org/project/kernel-tuner/0.4.5/#files), so that is I why I opted for building the conda-forge package for version 1.x. However, that requires `python-constraint2` which is only available as a [beta version](https://pypi.org/project/python-constraint2/#history), I suspect this is because maintenance was recently taken up again by a new maintainer, @fjwillemsen.